cirq/testing/json.py

```diff
@@ -0,0 +1,38 @@
+import io
+import numpy as np
+import pandas as pd
+
+import cirq
+
+
+def assert_roundtrip(obj, text_should_be=None, resolvers=None):
```

Comment:
This name is too vague; there is not enough context to know what `cirq.testing.assert_roundtrip` refers to.

Rename this to something more explicit like `assert_json_roundtrip_works`.

We may want to include this into the standard test battery for gates (`assert_implements_consistent_protocols`).

Reply:
Fixed the name, thanks!

I tried adding this to assert_implements_consistent_protocols, but that function is called with a ton of test gates (e.g. `GateUsingWorkspaceForApplyUnitary`) :\